### PR TITLE
Repair Drone Permissions Link

### PR DESCRIPTION
**Pull request by eichmeierbr**

### CODE REVIEW

File: _data/navigation.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@ wiki:
   - title: Project Management
     url: /wiki/project-management/
     children:
+      - title: Drone Flight Permissions
+        url: /wiki/system-design-development/drone-flight-permissions/
```
Comment (smbryan) [inline _data/navigation.yml:30]: This is a good entry for Project Management, but doesn't match the file location. It should be `wiki/project-management/drone-flight-permissions/`

**Comment by smbryan** [comment]
Solid submission. One fix, then you're in.

File: _data/navigation.yml (hunk 1 above)
Comment (eichmeierbr) [reply _data/navigation.yml]: Thank you, good catch. I made the fix as requested.

**Comment by smbryan** [comment]
Fastest turnaround I've seen! Thanks.

**Pull request merged**